nhrpd/vici.c

```diff
@@ -550,7 +550,7 @@ int sock_open_unix(const char *path)
 
 	memset(&addr, 0, sizeof(struct sockaddr_un));
 	addr.sun_family = AF_UNIX;
-	strncpy(addr.sun_path, path, sizeof(addr.sun_path) - 1);
+	strlcpy(addr.sun_path, path, sizeof(addr.sun_path) - 1);
```

Comment:
I think `sizeof(...) - 1` is no longer necessary in `strlcpy(addr.sun_path, path, sizeof(addr.sun_path) - 1);`. That idiom only works for `strncpy` with destination buffers zeroed out.